src/main/java/io/cdap/e2e/utils/SeleniumDriver.java

```diff
@@ -77,7 +78,7 @@ public static void getWaitDriver() {
 
   public static void openPage(String url) {
     logger.info(url);
-    logger.info(chromeDriver);
+    logger.info(String.valueOf(chromeDriver));
```

Comment:
why is this change needed?

Reply:
log4j's logger.info was accepting object as param, whereas logback is not